src/datepicker/datepicker.js

```diff
@@ -76,7 +77,12 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
   }
 
   $scope.isActive = function(dateObject) {
-    if (self.compare(dateObject.date, self.activeDate) === 0) {
+    var compareDate = typeof self.otherMonthSelectedDate !== 'undefined' ? self.otherMonthSelectedDate : self.activeDate;
```

Comment:
Change this check - one should use `angular.isUndefined`, or a more appropriate check as necessary.